src/pages/documentation/edge-network/release-notes.md

```diff
@@ -8,6 +8,17 @@ keywords:
 
 # Release Notes
 
+## September 1, 2023
+
+### iOS Edge 4.2.0
+
+* Added the ability to request a "completion" event when making an experience event request. The `contentComplete` event is dispatched after the request connection is closed.
+* Fixed record separator logic for iOS 17 and later.
```

Comment:
@timkimadobe was this a test change rather than sdk code change? if so we can drop from the release notes